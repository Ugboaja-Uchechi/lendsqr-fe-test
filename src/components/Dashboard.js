import React from "react";
import Header from "../subComponents/Header";
import SideBar from "../subComponents/sideBar";

const Dashboard = () => {
  return (
    <>
      <Header />
      <div className="flex-container">
        <SideBar />
        <h1 style={{ textAlign: "center", width: "100%" }}>
          Welcome to LendSqr
        </h1>
      </div>
    </>
    
  )
}

export default Dashboard;