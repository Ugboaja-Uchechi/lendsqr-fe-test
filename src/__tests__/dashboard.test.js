import React from "react";
import renderer from "react-test-renderer";
import Dashboard from "../components/Dashboard";

it('matches quote snapshot', () => {
  const dashboard = renderer.create(
    <Dashboard />,
  ).toJSON();
  expect(dashboard).toMatchSnapshot();
})