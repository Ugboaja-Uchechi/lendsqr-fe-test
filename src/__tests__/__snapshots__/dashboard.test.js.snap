// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches quote snapshot 1`] = `
Array [
  <header>
    <div
      className="flex"
    >
      <div>
        <img
          alt="LendSqr Logo"
          className="logo"
          src="logo.svg"
        />
      </div>
      <div
        className="grid"
      >
        <input
          className="text"
          placeholder="Search for anything"
          type="text"
        />
        <span>
          <svg
            className="search-icon"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 1024 1024"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0 0 11.6 0l43.6-43.5a8.2 8.2 0 0 0 0-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
            />
          </svg>
        </span>
      </div>
    </div>
    <div
      className="flex"
    >
      <h4>
        Docs
      </h4>
      <svg
        className="bell"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 16 16"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M8 16a2 2 0 0 0 2-2H6a2 2 0 0 0 2 2zM8 1.918l-.797.161A4.002 4.002 0 0 0 4 6c0 .628-.134 2.197-.459 3.742-.16.767-.376 1.566-.663 2.258h10.244c-.287-.692-.502-1.49-.663-2.258C12.134 8.197 12 6.628 12 6a4.002 4.002 0 0 0-3.203-3.92L8 1.917zM14.22 12c.223.447.481.801.78 1H1c.299-.199.557-.553.78-1C2.68 10.2 3 6.88 3 6c0-2.42 1.72-4.44 4.005-4.901a1 1 0 1 1 1.99 0A5.002 5.002 0 0 1 13 6c0 .88.32 4.2 1.22 6z"
        />
      </svg>
      <img
        alt="A girl wearing glasses laughing"
        className="avatar"
        src="avatar.png"
      />
      <p>
        Adedeji
      </p>
      <svg
        className="dropdown"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M7 10l5 5 5-5z"
        />
      </svg>
    </div>
  </header>,
  <div
    className="flex-container"
  >
    <svg
      className="menu"
      fill="currentColor"
      height="1em"
      onClick={[Function]}
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 16 16"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M16 5H0V4h16v1zm0 8H0v-1h16v1zm0-4.008H0V8h16v.992z"
      />
    </svg>
    <section
      className="side-bar"
      onClick={[Function]}
    >
      <div
        className="flex p-top"
      >
        <div>
          <img
            alt="A briefcase icon"
            className="img"
            src="briefcase.svg"
          />
        </div>
        <div>
          <h4>
            Switch Organisation
          </h4>
        </div>
        <svg
          className="dropdown-img"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </div>
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="home.svg"
          />
        </div>
        <div>
          <a
            href="/dashboard"
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Dashboard
            </h4>
          </a>
        </div>
      </div>
      <h3>
        Customers
      </h3>
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="user-friends.svg"
          />
        </div>
        <div>
          <a
            href="/users"
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Users
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="users.png"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Guarantors
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="sack.png"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Loans
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="handshake.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Decision Models
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="piggy-bank.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Savings
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="loan.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Loan Requests
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="whitelist.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Whitelist
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="karma.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Karma
            </h4>
          </a>
        </div>
      </div>
      <h3>
        Businesses
      </h3>
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="briefcase.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Organisation
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="loan.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Loan Products
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="bank.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Savings Products
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="coins.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Fees and Charges
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="transaction.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Transactions
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="services.png"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Services
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="user-cog.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Service Account
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="scroll.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Settlement
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="chart-bar.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Reports
            </h4>
          </a>
        </div>
      </div>
      <h3>
        Settings
      </h3>
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="sliders.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Preferences
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="fees.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Fees and Pricing
            </h4>
          </a>
        </div>
      </div>
      <h3 />
      <div
        className="flex m-top"
      >
        <div>
          <img
            alt="Icon"
            className="img"
            src="clipboard.svg"
          />
        </div>
        <div>
          <a
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <h4>
              Audit Logs
            </h4>
          </a>
        </div>
      </div>
      <h3 />
    </section>
    <h1
      style={
        Object {
          "textAlign": "center",
          "width": "100%",
        }
      }
    >
      Welcome to LendSqr
    </h1>
  </div>,
]
`;
